protocols/directdefund/directdefund.go

```diff
@@ -325,3 +333,23 @@ type ObjectiveRequest struct {
 func (r ObjectiveRequest) Id(myAddress types.Address) protocols.ObjectiveId {
 	return protocols.ObjectiveId(ObjectivePrefix + r.ChannelId.String())
 }
+
+func getPayload(b []byte) state.SignedState {
```

Comment:
godoc and maybe rename to `getSignedStatePayload`

Reply:
https://github.com/statechannels/go-nitro/pull/864/commits/38fc864269bc3d91827ebdbf9e203f2312db572a